src/pudl/io_managers.py

```diff
@@ -570,10 +570,18 @@ def __init__(self, base_path: UPath, schema: pa.Schema) -> None:
 
     def dump_to_path(self, context: OutputContext, obj: pd.DataFrame, path: UPath):
         """Write dataframe to parquet file."""
-        raise NotImplementedError("This IO Manager doesn't support writing data.")
+        logger.info(f"Write df to parquet at {path}")
+        schema = Resource.from_id("hourly_emissions_epacems").to_pyarrow()
```

Comment:
You should be able to get the asset name from the `context` object instead of hardcoding cems here. See `pudl.io_managers.SQLiteIOManager._get_table_name()` for how to do that.